keyboards/pancake/feather/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
+F_CPU = 8000000
+
+#
```

Comment:
Lines 7-15 and 19-21 can be deleted. QMK now configures defaults for these rules based on the defined MCU, which can be overridden.

https://github.com/qmk/qmk_firmware/blob/0b89809ac4e2ff0e84f9e4cef768c18165ef38ba/quantum/mcu_selection.mk#L39-L75

Reply:
4 to 15*

Reply:
@fauxpark `F_CPU` here is a different value from the QMK default (8 MHz instead of 16).